src/parser/xasl_generation.c

```diff
@@ -18214,12 +18227,22 @@ pt_to_insert_xasl (PARSER_CONTEXT * parser, PT_NODE * statement)
 	  (void) memcpy (xasl->class_locks + 1, aptr->class_locks, sizeof (int) * aptr->n_oid_list);
 	  (void) memcpy (xasl->tcard_list + 1, aptr->tcard_list, sizeof (int) * aptr->n_oid_list);
 
+	  xasl->includes_tde_class = aptr->includes_tde_class;
```

Comment:
It seems to have to be changed as above
```
if (aptr->includes_tde_class == 1) 
   xasl->includes_tde_class = aptr->includes_tde_class;
```

Reply:
Thanks.
Reflected in https://github.com/CUBRID/cubrid/pull/2450/commits/3627e56ff5f6b0225081f1376375b7ae23a7d6b3.